solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -443,16 +449,21 @@ contract RandomBeacon is Ownable {
     function genesis() external {
         require(groups.numberOfActiveGroups() == 0, "not awaiting genesis");
 
+        createGroupWarmUp();
+
         createGroup(
             uint256(keccak256(abi.encodePacked(genesisSeed, block.number)))
         );
     }
 
+    /// @notice Warms up the group creation process.
+    function createGroupWarmUp() internal {
```

Comment:
Can't we call `dkg.warmUp()` (or `dkg.lockState()` - see other comment) directly? We are not really creating anything here :shrug:

Reply:
We need to call it directly in tests via `publicCreateGroupWarmUp`. Calling `dkg.warmUp` directly in the stub causes a compilation error.

Reply:
Let's then name it the same as `dkg.` function named, document in `@dev` we need it for tests and move to the bottom of `internal` functions.

And one day, let's get rid of the stub exposing `publicCreateGroupWarmUp` completely... 🙏🏻 🤞🏻

Reply:
Done as part of https://github.com/keep-network/keep-core/pull/2691/commits/a29a5ad1f24125d53b37cef20129d6c5977764be